libshaderc_util/include/libshaderc_util/compiler.h

```diff
@@ -76,8 +75,8 @@ class Compiler {
 
   // Adds an implicit macro definition obeyed by subsequent CompileShader()
   // calls.
-  void AddMacroDefinition(const string_piece& macro,
```

Comment:
Should say the second argument can be nullptr, and how that's interpreted.
Add a test for that case.